public/css/common.less

```diff
@@ -151,17 +159,25 @@ div.show-more {
     // Icinga Web 2 override
     .spinner {
       height: 100%;
-      padding-right: 0;
+    }
+
+    .control-button {
+      margin: 0;
+    }
+
+    .link-button {
```

Comment:
Isn't that the `.control-button` now? Or is this used somewhere else?

Reply:
You're right. Dropped that block completely, since it's not used anywhere else.

Reply:
✅